addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/EvohomeApiClient.java

```diff
@@ -17,4 +18,5 @@
 
     ControlSystem getControlSystem(int id);
 
-}
\ No newline at end of file
+    ZoneStatus getHeatingZone(int locationId, int zoneId);
```

Comment:
Yeah I decided to try and go fully for V2 to stay with you - especially as you'd done the hard bit of calling the requests. You are right though shouldn't be on the interface.